Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -348,9 +348,7 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
     static WTF::String defaultAlternativeServicesDirectory();
     static WTF::String defaultApplicationCacheDirectory();
     static WTF::String defaultWebSQLDatabaseDirectory();
-#if USE(GLIB)
     static WTF::String defaultHSTSDirectory();
```

Comment:
Do we need to wrap this with `#if USE(GLIB) || PLATFORM(COCOA)`?

Reply:
Yes.

Reply:
I would remove the guards, and add a dummy implementation in WebsiteDataStore.cpp instead. Of course either way is fine.